javaschool-backend/src/main/java/com/nearsoft/javaschoolbackend/model/response/PackageSize.java

```diff
@@ -0,0 +1,34 @@
+package com.nearsoft.javaschoolbackend.model.response;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org/](url)